scripts/Muon/table_utils.py

```diff
@@ -72,4 +72,5 @@ def setTableHeaders(table):
                 +"background-color:white;"\
```

Comment:
A few lines up line 56.  Is it possible to use the QT constant here  QSysInfo::WV_WINDOWS10 rather than the magic number?

Reply:
the command is slightly different for python. I am compiling it now to check it works on windows 10